assemblies/assembly_release-notes/main.adoc

```diff
@@ -21,9 +21,10 @@ include::{enterprise-dir}/release-notes/rn-tech-preview-ref.adoc[leveloffset=+1]
 
 include::{enterprise-dir}/release-notes/rn-known-issues-ref.adoc[leveloffset=+1]
 
-include::{enterprise-dir}/release-notes/rn-7.1.0-fixed-issues-ref.adoc[leveloffset=+1]
+include::{enterprise-dir}/release-notes/rn-720-fixed-issues-ref.adoc[leveloffset=+1]
+//include::{enterprise-dir}/release-notes/rn-7.1.0-fixed-issues-ref.adoc[leveloffset=+1]
 
-include::{enterprise-dir}/release-notes/rn-7.1.1-fixed-issues-ref.adoc[leveloffset=+1]
+//include::{enterprise-dir}/release-notes/rn-7.1.1-fixed-issues-ref.adoc[leveloffset=+1]
 
 
```

Comment:
Things not in the PR:
* Under "Experimental properties administration page": "administration page, set" no space before comma
* Under "Enhanced process instance ....": Should probably be "node-entered events" and "node-completed ones", also should be "such as ID, ...".
* **Important:** After the above section, the "Support for SSH Keystore" section is not rendered correctly in the output.
* Under "Decision Model and Notation (DMN) models": I thought I updated the language to specify that the DMN designer was tech preview, but I don't see that reflected for some reason. Easiest to probably just add this note as shown below, consistent with what I have in the DMN doc. Would you mind adding this?
```
... projects for deployment and execution.

NOTE: The DMN designer in {PRODUCT} {PRODUCT_VERSION} is a Technology Preview feature and is disabled by default in {CENTRAL}. To enable the DMN designer preview in {CENTRAL}, in the upper-right corner of the window, click *Settings* -> *Roles*, select a role from the left panel, click *Editors* -> *DMN Designer* -> *Read*, and click *Save* to save the changes.

For more information ...
```
* Under "Defining weights ...": Should be "In {PLANNER}, " Right now it says "In Business Planner," which is nothing :) 
* Under "Business Central compensation ...": "to roll back the actions"; for last sentence in paragraph ("There are"), line break and change to "Three types of compensation events are available in {CENTRAL}:"; and change to "end (throw) events:" with colon.